tests/QssTestDefs.js

```diff
@@ -111,6 +111,75 @@ gpii.tests.qss.testPspAndQssVisibility = function (app, params) {
     );
 };
 
+gpii.tests.qss.getFocusedElementIndex = function () {
+    // Note that the elements will be returned in the order in which they appear in the DOM.
+    var qssButtons = jQuery(".fl-qss-button"),
+        focusedElement = jQuery(".fl-focused")[0];
+    return jQuery.inArray(focusedElement, qssButtons);
+};
+
+gpii.tests.qss.pressKey = function (key, modifiers) {
+    return {
+        funcName: "gpii.tests.qss.simulateShortcut",
+        args: [
+            "{that}.app.qssWrapper.qss.dialog",
+            {
+                key: key,
+                modifiers: modifiers
+            }
+        ]
+    };
+};
+
+gpii.tests.qss.assertFocusedElementIndex = function (expectedIndex) {
+    return {
+        task: "gpii.test.executeJavaScriptDelayed",
+        args: [
+            "{that}.app.qssWrapper.qss.dialog",
+            gpii.test.toIIFEString(gpii.tests.qss.getFocusedElementIndex),
+            100
+        ],
+        resolve: "jqUnit.assertEquals",
+        resolveArgs: [
+            "The correct button in the QSS is focused",
+            expectedIndex,
+            "{arguments}.0"
+        ]
+    };
+};
+
+var qssSettingsCount = 11;
+
+var navigationSequence = [
+    {
+        func: "{that}.app.tray.events.onTrayIconClicked.fire"
+    },
+    gpii.tests.qss.assertFocusedElementIndex(-1),
```

Comment:
Probably a comment might be useful for edge cases such as this one.
Additionally it would be better if `Left` key press when no item is focused is also tested.